src/_help/hubs/index.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Hubs
+---
+
+- TOC
+{:toc}
+
+## What is a hub
+
+Hubs are a feature available starting from our paid plans: they allow you to manage collections of API documentation simply and efficiently.
+
+With hubs, your API ecosystem is displayed on a single page, where documentation can be sorted by categories.
+
+Hubs also provide a unified changelog, making it easy to identify all changes across all the documentation it contains at a glance. Search is also global, allowing you to search across all documentation.
+
+## Create a hub
+
+In your dashboard, click on Hubs then on Create a hub.
+
+![](/images/help/create-hub.png)
+
+You will be asked to choose a name for the hub, which will be visible to everyone. The slug is the portion of the URL that you can customize but you can also choose to use a custom domain instead:
+- By default, on Bump.sh, your hub will be available on this URL https://bump.sh/hub/<your-own-slug>
+- With a custom domain, your hub will be available on https://www.custom.domain.com
+
+Finally, you can decide whether this hub is visible to everyone or only to members of your organization.
+
+![](/images/help/hub-creation.png)
+
+## Add documentation to a hub
+
+### Using the webapp
+#### Deploy a new documentation
+From the "Hubs" tab, click on "Create Documentation" to add it to the desired hub. You can then specify the documentation's name, its slug, and its access level (Public or private).
+During the creation of a stand-alone documentation, you'll also find the option to add it automatically to an existing hub.
+#### Transfer an existing documentation
+From any documentation, you can choose to move it to an existing hub. The "Hubs Settings" section allows you to choose which hub to associate this documentation with. You can also specify a category if necessary.
+
+![](/images/help/move-standalone-doc.png)
+### Using the CLI
```

Comment:
I would add a warning that you can't create a new doc without deploying a first version via the CLI (and also via the API)

Reply:
Also, that you can't customize the doc `name` and `description` via the CLI or the API